docker-for-windows/troubleshoot.md

```diff
@@ -286,43 +277,37 @@ script](https://github.com/moby/moby/issues/24388).
 
 ### Virtualization
 
-In order for Docker Desktop for Windows to function properly your machine needs:
+In order for Docker Desktop to function properly your machine needs:
```

Comment:
This section is prereqs for virtualization, so I suggest changing "In order for Docker Desktop to function properly" to "For Docker Desktop to function correctly, you machine must meet (or have) the following components (features, products, ?) installed:"

Reply:
Fixed in commit 59aa7e5.